doc/fork_support.md

```diff
@@ -0,0 +1,49 @@
+# Background #
+
+In Python multithreading is ineffective at concurrency for CPU bound tasks
+due to the GIL.
+(global interpreter lock).  Users use multiprocessing, subprocess, 
+and concurrent.futures.ProcessPoolExecutor, to work around the GIL.
```

Comment:
nit: no comma after `ProcessPoolExecutor`, and we may not want this to read like an exhausive list (e.g., "Users use libraries such as...")

Reply:
Done